File: PHP/UberBlack.php
<?php
require_once("Car.php");
class UberBlack extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        
        parent::__construct($license, $driver); //construccion de la clase padre (superclase)
        
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

?>

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

// Aplicar herencia a esta clase. Hereda de Car
public class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    // Constructos que recibe 2 datos
    public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(license, driver); //super representa a la super clase

        // this hace referencia a los atributos de la clase en la que me encuentro en este archivo
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}